src/task.c

```diff
@@ -510,6 +510,8 @@ JL_NO_ASAN static void ctx_switch(jl_task_t *lastt)
     jl_set_pgcstack(&t->gcstack);
     jl_signal_fence();
     lastt->ptls = NULL;
+    fegetenv(&lastt->fenv);
```

Comment:
This should come before the task switch, as some task switch implementations will clobber it

Reply:
Where specifically should it go?

Reply:
Or rather, it looks like the fesetenv is the one that needed to move

Reply:
Where should it go?

Reply:
Feel free to push to this branch

Reply:
Already done. I don't know about the propagating behavior though. I think we might want to start every task with a call to set `fesetenv(FE_DFL_ENV)`?